jakarta.ls/src/main/java/org/eclipse/lsp4jakarta/commons/SnippetRegistry.java

```diff
@@ -172,7 +173,8 @@ public List<CompletionItem> getCompletionItem(final Range replaceRange, final St
             String label = snippet.getPrefixes().get(0);
             CompletionItem item = new CompletionItem();
             item.setLabel(label);
-            item.setDetail(snippet.getDescription());
+//            item.setDetail(snippet.getDescription());
```

Comment:
Looks like we can safely remove this line as well

Reply:
Keep the line there in order to have some other thoughts for JSON files.